src/managers/contact_manager.py

```diff
@@ -20,28 +20,49 @@
 """
 
 import re
+import os
```

Comment:
Hi @MHryhoriev !
Can we remove unused import ?

Reply:
I may have missed this point, but in fact, such cases will be removed later, when we complete the main development and check the project for such nuances.